lib/ui/surveydetail/survey_detail_screen.dart

```diff
@@ -126,16 +150,116 @@ class SurveyDetailScreenState extends ConsumerState<SurveyDetailScreen> {
         : const SizedBox.shrink();
   }
 
+  Widget _buildActionButton(BuildContext context, int index) {
+    if (index == 0) {
+      return _buildStartSurveyButton(context);
+    } else {
+      final isLastPage = index == pages.length;
+      if (isLastPage) {
+        return _buildSubmitButton();
+      } else {
+        return _buildNextButton();
+      }
+    }
+  }
+
+  Widget _buildStartSurveyButton(BuildContext context) {
+    return TextButton(
+      key: SurveyDetailScreenKey.btStart,
+      style: ButtonStyle(
+        backgroundColor: MaterialStateProperty.all(Colors.white),
+        foregroundColor: MaterialStateProperty.all(Colors.black),
+        overlayColor: MaterialStateProperty.all(Colors.black12),
+        padding: MaterialStateProperty.all(
+          const EdgeInsets.symmetric(
+            vertical: 19,
+          ),
+        ),
+        shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+          RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+        ),
+        textStyle: MaterialStateProperty.all(
+          Theme.of(context).textTheme.labelLarge,
+        ),
+      ),
+      onPressed: _gotoNextPage,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 20),
+        child: Text(AppLocalizations.of(context)!.startSurvey),
+      ),
+    );
+  }
+
+  Widget _buildNextButton() {
+    return Padding(
+      padding: const EdgeInsets.only(left: 20),
+      child: GestureDetector(
+        key: SurveyDetailScreenKey.btQuestionNext,
+        onTap: _gotoNextPage,
+        child: ClipOval(
+          child: Material(
+            color: Colors.white,
+            child: SizedBox(
+              width: 56,
+              height: 56,
+              child: Assets.icons.icArrowRight.svg(
+                fit: BoxFit.none,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildSubmitButton() {
+    return TextButton(
+      key: SurveyDetailScreenKey.btQuestionSubmit,
+      style: ButtonStyle(
```

Comment:
Same here

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/58/commits/c3e28b47d8e45884f706677f35777c7eb264e6f0